<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Warehouse</class>
 <widget class="QWidget" name="Warehouse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Almac√©n central</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="utils.qrc">
         <normaloff>:/icons/search_48.png</normaloff>:/icons/search_48.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="styleSheet">
      <string notr="true">selection-color: rgb(255, 255, 255);
selection-background-color: rgb(85, 170, 255);</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="inputButton">
       <property name="text">
        <string>Entrada</string>
       </property>
       <property name="icon">
        <iconset resource="utils.qrc">
         <normaloff>:/icons/box_download_48.png</normaloff>:/icons/box_download_48.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outputButton">
       <property name="text">
        <string>Salida</string>
       </property>
       <property name="icon">
        <iconset resource="utils.qrc">
         <normaloff>:/icons/box_upload_48.png</normaloff>:/icons/box_upload_48.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>searchEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>inputButton</tabstop>
  <tabstop>outputButton</tabstop>
 </tabstops>
 <resources>
  <include location="utils.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>setfilter()</slot>
 </slots>
</ui>
